cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -181,6 +191,16 @@ private static boolean parseCreateClusterRoles(String createClusterRolesEnvVar)
         return createClusterRoles;
     }
 
+    private static boolean parseRolesOnly(String rolesOnlyEnvVar) {
+        boolean rolesOnly = DEFAULT_STRIMZI_ROLES_ONLY;
+
+        if (rolesOnlyEnvVar != null) {
+            rolesOnly = Boolean.parseBoolean(rolesOnlyEnvVar);
```

Comment:
Probably we should validate that the value is either `true` or `false` and error if it's anything else.